registry/remote/repository.go

```diff
@@ -828,7 +837,7 @@ func (s *manifestStore) Resolve(ctx context.Context, reference string) (ocispec.
 
 // FetchReference fetches the manifest identified by the reference.
 // The reference can be a tag or digest.
-func (s *manifestStore) FetchReference(ctx context.Context, reference string) (desc ocispec.Descriptor, rc io.ReadCloser, err error) {
+func (s *manifestStore) FetchReference(ctx context.Context, reference string) (ocispec.Descriptor, io.ReadCloser, error) {
```

Comment:
The return parameters are named for their special purposes. Removing the names will introduce bugs in line 858-862.